core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/models.scala

```diff
@@ -60,23 +59,18 @@ object StagingBucket {
 
 /**
  * Can't extend final enum State from WSM, so made a wrapper
- * WSM state can be BROKEN, CREATING, DELETING, READY, UPDATING or None
- * if None --> it is deletable and is deleted
- * (already deleted in WSM, need to clean up leo resources)
+ * WSM state can be Some(BROKEN, CREATING, DELETING, READY, UPDATING) or None
+ * if None --> there is no record of this resource in WSM
+ * (already deleted or never existed in WSM, need to clean up leo resources)
  */
 case class WsmState(state: Option[String]) {
 
-  val deletableStatuses: Set[String] = Set("BROKEN", "READY", "DELETED")
-  val possibleStatuses: Array[String] = State.values().map(_.toString) :+ "DELETED"
+  val deletableStatuses: Set[String] = Set("BROKEN", "READY", "NONE")
```

Comment:
switched this from "DELETED" to "NONE" to more accurately reflect the option that the resource could've never been created

Reply:
Also, `DELETED` does not really exist anyway since the record get's deleted from the table?